src/main.rs

```diff
@@ -130,6 +138,10 @@ enum TopLevelItem {
         name: String,
         body: TopLevel,
     },
+    Type {
```

Comment:
```suggestion
    TypeDefinition {
```
to differentiate it from the notion of `Type`